scripts/media-player

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
+# for storing possible user defined arguments for playerctl
+PLAYER_ARGS=$(xrescat i3xrocks.media_player.player_args)
```

Comment:
I want to verify, this will also work if `i3xrocks.media_player.player_args` is undefined or should it default to `status`.  If so you can just add `status` like so:
```
$(xrescat i3xrocks.media_player.player_args status)
```

Reply:
I tested it out and it still works as usual aka taking the first player in the `playerctl -l` list if there's no Xresource defined. If the user puts some random mumbo jumbo to it then the script just dies but that's on the user.

The actual `status` command is executed on the the next line nevertheless of the given player args:
```bash
STATUS=$(playerctl $PLAYER_ARGS status 2> /dev/null)
```